### ISSUE TITLE
Create linked-tabs widget Asciidoctor directive

### ISSUE THREAD

**Issue by gtback** [elastic/docs#1896]
elastic/beats#17007 added lots of custom pass-through HTML to support linked tab widgets.

Add an Asciidoctor extension/directive to make it easier to create these on a page.

Depends on #1895

**Comment by gtback** [comment]
This is really a duplicate of #260
